projects/schematics/src/shared/utils/lib-utils.ts

```diff
@@ -248,17 +249,20 @@ export function addLibraryFeature<T extends LibraryOptions>(
   options: T,
   config: FeatureConfig
 ): Rule {
-  return (tree: Tree) => {
+  return (tree: Tree, context: SchematicContext) => {
     const spartacusFeatureModuleExists = checkAppStructure(
       tree,
       options.project
     );
     if (!spartacusFeatureModuleExists) {
-      throw new SchematicsException(
+      context.logger.warn(
         'Please migrate manually to new app structure: https://sap.github.io/spartacus-docs/reference-app-structure/ and add the library once again. Old app structure is no longer supported.'
       );
+      context.logger.info('Scaffolding the new app structure...');
```

Comment:
Can we change messages here?
First I would inform that we are scaffolding new app structure and then I would warn customer to migrate his modules to reference app structure we created. We can now remove the message that the old structure is not supported